Cattac/Cattac/GameEngine.swift

```diff
@@ -26,14 +26,17 @@ class GameEngine {
 
     // The AI engine that is used when multiplayer mode is not active
     private var gameAI: GameAI!
+    
+    /// States to advance, initialized at 1 to rollover PreStart state.
+    private var statesToAdvance: Int = 1
 
     var gameManager: GameManager = GameManager()
     
     /// Player index (we should change to player-id instead).
     var playerNumber = 1
     
-    // The initial game state is to be set at precalculation
-    var state: GameState = .Precalculation
+    // The initial game state is to be set at Initialization
+    var state: GameState = .PreStart
```

Comment:
You can set the gameState to `PostExecute` lol

I don't think you need to worry too much on the 60 times per second, that's mainly for the animations states `MovementExecution` and `ActionsExecution`. Both of which does nothing in the switch case block.